tests/functional/test_to_idp_interstitial.py

```diff
@@ -0,0 +1,37 @@
+import time
+
+from flask import url_for
+import pytest
+
+
+@pytest.mark.usefixtures('live_server')
+class When_on_to_idp_interstitial(object):
+
+    @pytest.fixture(autouse=True)
+    def setup_page(
+            self, browser, config, email_address, submit_known_email_address):
+        self.oidc_client_issuer = config.get('OIDC_CLIENT')['issuer']
```

Comment:
This `config` should be `app.config` - pass `app` instead of `config` as a parameter